ext/datasources/helium/helium.c

```diff
@@ -2440,6 +2430,13 @@ helium_session_truncate(WT_DATA_SOURCE *wtds,
 
 	ESET(unlock(wt_api, session, &ws->lock));
 	return (ret);
+#else
+	(void)wtds;
+	(void)session;
+	(void)uri;
+	(void)config;
+	return (0);
```

Comment:
This is the truncate code, I expect to fix it in one way or another before this branch is merged.